app/src/test/kotlin/maru/testutils/MaruFactory.kt

```diff
@@ -49,7 +49,7 @@ object MaruFactory {
     endpoint = "$ethereumJsonRpcUrl"
 
     [qbft-options]
-    communication-margin=200m
+    communication-margin=10m
```

Comment:
I believe that's not enough to send and process requests. Do we need this delay at all?